tests/test_federal_senate_dataset.py

```diff
@@ -1,41 +1,102 @@
 import os
 from tempfile import gettempdir
-from unittest import main, skipIf, TestCase, TestLoader
+from unittest import TestCase
+from unittest.mock import patch
+
+import pandas as pd
 
 from serenata_toolbox.federal_senate.federal_senate_dataset import FederalSenateDataset
 
+
 class TestFederalSenateDataset(TestCase):
-    def setUp(self):
+    @classmethod
+    def setUpClass(cls):
+        cls.expected_files = ['federal-senate-2008.csv',
+                              'federal-senate-2009.csv']
+
+    @patch('serenata_toolbox.federal_senate.federal_senate_dataset.urlretrieve')
+    def test_fetch_files_from_S3(self, mockedUrlRetrieve):
```

Comment:
Addressed that in 598545a0f1c479503fc6c6bd99c6c34cdd496801, feel free to bring that to your branch ; )